### PR TITLE
Move ilContObjParser to Test and Rename

### PR DESCRIPTION
**Pull request by kergomard**
I @alex40724 
I would suggest to move as by your suggestion in [this comment](https://github.com/ILIAS-eLearning/ILIAS/pull/5097#issuecomment-1270238258) and move the `ilContParser` class to the test. OK?

### CODE REVIEW

**Review by alex40724 — APPROVED** [review]
Great! Thank you very much.

**Comment by alex40724** [comment]
You may merge this anytime if ok for the T&A maintainers.

**Comment by dsstrassner** [comment]
This is fine for me. Thanks!

**Pull request merged**
